nemo/collections/nlp/models/dialogue_state_tracking_hybrid/sgdqa_model.py

```diff
@@ -0,0 +1,623 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
```

Comment:
Could you make sure this ckpt compatible with the new code? https://ngc.nvidia.com/models/nvstaging:nemo:sgdqa_bertbasecased

Reply:
This shouldn't be a problem as I only changed the Dataset class and .nemo doesn't save the dataset information if I understand it correctly. Nonetheless, if you would like me to double check this, can you send a link on how I can check for this (not too sure how I can load the model from ngc) ?